ci/purge.ps1

```diff
@@ -0,0 +1,29 @@
+# This script is used to kill all dangling procceses in the pre-exit hook.
+
+$SCRIPT_LOCATION = $MyInvocation.MyCommand.Path
+$SCRIPT_DIR = Split-Path $SCRIPT_LOCATION
+cd "${SCRIPT_DIR}/.."
+
+function Kill-Dangling-Processes {
+	param( [string]$ProcessName )
+
+	echo "Looking for ${ProcessName} processes to kill..."
```

Comment:
Why not use the `Get-Process` cmdlet? https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-process?view=powershell-6
That gives you back objects, and means you don't have to do regex and string splitting.

Reply:
The idea is to get processes handles that are locking files in the build directory - from a cursory look I don't think `Get-Process` supports that behaviour.

Reply:
Aha, that intent isn't clear from comments or obvious command flags

Reply:
Will add some comments about what this script does 👍

Reply:
PTAL 😄